### ISSUE TITLE
DatePicker 400% Zoom not working

### ISSUE THREAD

**Issue by jazzAlston** [microsoft/fluentui#18902]
Browser and OS versions: Google Chrome /Windows 10
Describe the issue:
We browser zoom of 400% applied the DatePicker does not apply scrollbar.

https://codepen.io/jazz8603/pen/vYxGXoE

Actual behavior:
User cannot see the calendar when browser zoom of 400% is applied.

Expected behavior:
When DatePicker is zoomed to 400% user should be able to scroll to see all content

Documentation describing expected behavior
https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Resize_content

**Comment by gouttierre** [comment]
@jazzAlston - Thank you for reaching out to us.

@smhigley - Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @ling1726

**Comment by smhigley** [comment]
Datepicker and Calendar were already updated to always have an overlaid month picker (so no side-by-side day and month grids) at high zoom. It shouldn't need a scrollbar to use, since the width is well under 320px.

@jazzAlston could you provide a screenshot of what you're seeing at 400% zoom?
